CRM/Contribute/Form/CancelSubscription.php

```diff
@@ -207,17 +208,16 @@ public function postProcess() {
       // civicrm_contribution_recur.processor_id
       $cancelParams = ['subscriptionId' => $this->_subscriptionDetails->subscription_id];
       try {
-        $cancelSubscription = $this->_paymentProcessorObj->cancelSubscription($message, $cancelParams);
+        $propertyBag = new PropertyBag();
+        $propertyBag->setRecurProcessorID($this->_subscriptionDetails->subscription_id);
```

Comment:
@eileenmcnaughton Please add in here: `$propertyBag->setContributionRecurID($this->_subscriptionDetails->recur_id);` so we have the recurring contribution ID available as well.

Reply:
OK - let's worry about adding new parameters in a follow up - the goal of this PR is just to switch with parity